qcelemental/models/align.py

```diff
@@ -22,17 +22,15 @@ class AlignmentMill(ProtoModel):
     """
 
     shift: Optional[Array[float]] = Field(  # type: ignore
-        None, description="Translation array (3,) for coordinates.",
+        None, description="Translation array (3,) for coordinates."
```

Comment:
could you have some odd local black setting? I confirmed that running `make format` doesn't remove the trailing commas for me.

Reply:
I'm not sure about this.  I forked the most recent version of qcel yesterday and ran make format.

Reply:
Going to have to return to below to pass
```
    mirror: bool = Field(
        False, description="Do mirror invert coordinates?",
    )
```

Reply:
are you sure this doesn't work? my PR testing the linter on the existing with-commas code passed

Reply:
Yeah, I don't think so... not sure what could be wrong.  Every time I run make format it removes the commas from align.py.  I don't ever remember changing any black settings